component/policies.jsonnet

```
@@ -0,0 +1,23 @@
+local kyverno = import '../lib/kyverno.libsonnet';
+local common = import 'common.libsonnet';
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+
+local params = inv.parameters.kyverno;
+
+local clusterpolicies = std.map(
+  function(name) kyverno.ClusterPolicy(name) {
+    metadata+: {
+      name: name,
+      labels+: common.Labels,
+    },
+  } + com.makeMergeable(params.clusterpolicies[name]),
+  std.objectFields(params.clusterpolicies),
+);
```

Consider using [`com.generateResources()`](https://syn.tools/commodore/reference/commodore-libjsonnet.html#_generateresourcesresources_resourcefn) here so you get automatic support for dropping policies by setting `the-policy-I-dont-want: null` and similar QoL features:

 ```suggestion
local clusterpolicies = com.generateResources(
  params.clusterpolicies,
  function(name) kyverno.ClusterPolicy(name) {
    metadata+: {
      labels+: common.Labels,
    },
  },
);
```